docs/merchant-of-record/account-reviews.mdx

```diff
@@ -33,3 +33,53 @@ We look at:
 Credit card networks, e.g Visa/Mastercard, consider 0.7% of sales in chargebacks excessive. Exceeding it can lead to monitoring programs with high costs, penalties and ultimately termination.
 
 We therefore reach out proactively to collaborate on maintaining a low chargeback ratio and reducing it ahead of getting close to these thresholds.
+
+## Operational Guidelines
+
+To maintain platform integrity and ensure smooth operations, we have established clear guidelines for merchants using Polar.
+
+#### Expected Responsiveness
+
+We expect merchants to maintain high standards of customer support and responsiveness:
+
+- **Support Ticket Management**: Maintain a low rate of support tickets from end-users
+- **Merchant Communication**: When we include you in customer support communications, we require a response within 48 hours
+  - If no response is received within 48 hours, we will issue a warning
+  - If no response is received within 24 hours of the warning, we may issue refunds to affected customers
+- **Customer Service Quality**: We evaluate your customer service history and approach when determining appropriate actions
+
+#### Test Transactions
+
+To maintain platform security and prevent abuse:
+
+- **Use Sandbox Environment**: All testing must be conducted using our sandbox environment
+- **No Real Money Testing**: Test transactions with real money are not permitted and will be flagged for review
+- **High-Value Test Transactions**: Test transactions exceeding $100 will be escalated for internal review
+  - We will contact you before taking any action on your account
+  - These transactions may result in refunds to maintain platform integrity
+
+#### Chargeback Management
+
+We monitor chargeback rates closely to protect both merchants and customers:
+
+- **Acceptable Threshold**: We maintain a chargeback rate threshold of 0.4% for merchants
+- **Timeframe**: Chargebacks can be filed up to 120 days from the original transaction date
+- **Consequences of High Chargeback Rates**:
+  - **Payment Processing**: Future payments may be denied
+  - **Account Status**: Your account may be temporarily paused while maintaining dashboard access
+  - **Payout Management**: 
+    - Payouts may be paused until chargeback rates improve
+    - We work with merchants in good faith to resolve issues while maintaining platform security
+    - In cases of excessive risk, refunds may be issued to protect customers
+
+#### Policy Violations
+
+For merchants who violate our [acceptable use policies](/merchant-of-record/acceptable-use) (and don't have high chargeback rates):
+
+- **Immediate Action**: We will offboard merchants who violate our policies
+- **Payment Processing**: All payment processing will be blocked
+- **Payout Management**: Payouts will be paused pending review
+- **Resolution Process**:
```

Comment:
I would change this to:

```
- **Resolution Process**:
   - In case of strong suspicion of fraud or intentional abuse, we block the account immediately
   - We reach out merchants about the issue and give them 48h to respond
   - We pause future payments in the meantime if deemed necessary
   - We pause payouts during the resolution process
   - We strive to collaborate with merchants on the best possible path forward. Clear fraud or abuse, however, is immediately blocked.
   - We have to cancel subscriptions and refund payments made in violation of our acceptable use policies for compliance and risk, and strive to do so in collaboration with the merchant.
```